Serverless tracing integrations updates (#9916)
DataDog/documentation: content/en/serverless/distributed_tracing/_index.md

#### Java

The Datadog Lambda Library and tracing libraries for Java support:
- Correlation of Lambda logs and traces with trace ID and tag injection. See [Connecting Java logs and traces][11] for more details.
- Tracing HTTP requests invoking downstream Lambda functions or containers.
- Tracing dozens of additional out-of-the-box [Java][12] libraries.

> You can trace your serverless functions in Datadog with [Datadog's tracing libraries][6] or by [installing our AWS X-Ray integration][3]. If you are using [Datadog's tracing libraries][6], and need to connect multiple Lambda function traces in event-driven architectures, we recommend you augment your traces by configuring _both_ AWS X-Ray and Datadog APM tracing libraries as described [here][7].

Reviewer: ```suggestion
You can trace your serverless functions in Datadog with [Datadog's tracing libraries][6] or by [installing the AWS X-Ray integration][3]. If you are using [Datadog's tracing libraries][6], and you need to connect multiple Lambda function traces in event-driven architectures, it is recommended that you augment your traces by configuring _both_ AWS X-Ray and Datadog APM tracing libraries, as described in [the serverless trace merging documentation][7].
```
